src/components/application_manager/test/policies/policy_handler_test.cc

```diff
@@ -1019,7 +1026,8 @@ TEST_F(PolicyHandlerTest, SendOnAppPermissionsChanged) {
   // Check expectations
   EXPECT_CALL(app_manager_, application_by_policy_id(kPolicyAppId_))
       .WillOnce(Return(application));
-  EXPECT_CALL(*MockMessageHelper::message_helper_mock(),
+  EXPECT_CALL(*application, app_id()).WillOnce(Return(0u));
```

Comment:
@vlantonov Is this a real application class object? If no - please add prefix `mock_` to variable name